ihmc-communication/src/test/java/us/ihmc/communication/ROS2ToolsTest.java

```diff
@@ -1,12 +1,9 @@
 package us.ihmc.communication;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
-import perception_msgs.msg.dds.REAStateRequestMessage;
 import org.apache.commons.lang3.mutable.MutableInt;
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
-
+import perception_msgs.msg.dds.REAStateRequestMessage;
```

Comment:
Did you mean to add this import?

Reply:
I must've auto formatted and it moved things around